usb-gadget/hal/file_contexts

```diff
@@ -0,0 +1,3 @@
+# usb role switch service
+genfscon sysfs /sys/class/typec  				u:object_r:sysfs_usb_typec:s0
```

Comment:
it should be:
genfscon sysfs /class/typec  				u:object_r:sysfs_usb_typec:s0

and you need to move this line to genfs_contexts file.